src/NHibernate/Dialect/Dialect.cs

```diff
@@ -148,15 +148,18 @@ protected Dialect()
 		public static Dialect GetDialect()
```

Comment:
Shouldn't this one be obsoleted too ? How would it work without `Environment.Properties` ?

Reply:
I decided to split this into several PRs.